ui/src/api/ragQueryApi.ts

```diff
@@ -94,7 +94,10 @@ const suggestQuestionsQuery = async (
   );
 };
 
-type ChunkContents = string;
+interface ChunkContentsResponse {
+  text: string;
+  metadata: Record<string, any>;
```

Comment:
maybe just `string` or `string | number`?

Reply:
we can start with that